JRC Publications Repository.js

```diff
@@ -67,10 +67,21 @@ function getSearchResults(doc, checkOnly) {
 function detectWeb(doc, url) {
 	var type = ZU.xpath(doc, '//meta[@name="DC.type"]/@content');
 	if (type.length>0) {
-		if (mappingTable[type[0].value]) {
-			return mappingTable[type[0].value];
-		} else {//generic fallback
-			Z.debug('Unrecognized item type: ' + type[0].value);
+		if (mappingTable[type[0].textContent]) {
+			return mappingTable[type[0].textContent];
```

Comment:
Yes,that's what we _should_ do, though we don't really follow that rule strictly. Going forward, I think we should pay more attention to this, since it may start breaking in the future.